src/client.c

```diff
@@ -4454,6 +4454,53 @@ void CheckTeamStatus()
 	}
 }
 
+void SendSpecInfo()
+{
+	gedict_t *t, *p;
+	int cl;
+	char *tracking, *nick;
+
+	static double lastupdate = 0;
+
+	if (g_globalvars.time - lastupdate < 2)
```

Comment:
TBH it is pretty ugly, in an ideal world you should press TAB and the client inform the server what you interesting in such info and send it ONLY then, that's how +schowscroes do for client pings, server send information about ping of other clients only when you press +schowscores, it should be same for such information as well, but it would require deeper integration with server (mvdsv) and some additional work from you...